gostream/stream.go

```diff
@@ -254,25 +256,33 @@ func (bs *basicStream) processInputFrames() {
 				defer framePair.Release()
 			}
 
-			bounds := framePair.Media.Bounds()
-			newDx, newDy := bounds.Dx(), bounds.Dy()
-			if bs.videoEncoder == nil || dx != newDx || dy != newDy {
-				dx, dy = newDx, newDy
-				bs.logger.Infow("detected new image bounds", "width", dx, "height", dy)
+			var encodedFrame []byte
+
+			if frame, ok := framePair.Media.(*rimage.LazyEncodedImage); ok && frame.MIMEType() == utils2.MimeTypeH264 {
+				encodedFrame = frame.RawData() // nothing to do; already encoded
+			} else {
+				bounds := framePair.Media.Bounds()
```

Comment:
Maybe I am reading this wrong but I thought that `Bounds()` is not implemented for `H264` struct. What is `framepair.Media` here? Should we implement those getters?